inc/alert.class.php

```diff
@@ -216,6 +216,68 @@ public function rawSearchOptions()
         return $tab;
     }
 
+    // @codingStandardsIgnoreStart
+    public function post_updateItem($history = true)
+    {
+        // @codingStandardsIgnoreEnd
+        if (
+            !isset($this->input['is_close_allowed'])
+            || $this->input['is_close_allowed'] !== '0'
+        ) {
+            return;
+        }
+
+        $alertUserId = $this->getAlertUserId($this->input['id']);
+
+        if ($alertUserId !== null) {
+            $this->updateAlertUserState($alertUserId, 0);
```

Comment:
If I understand correctly, this is going to find the first `Alert_User` item for the current alert and update it.

Shouldn't we update ALL  `Alert_User` items for the current alert instead (since the goal is to make them visible again) ?

Reply:
Thanks for the comment, I forgot to correct some of the code. :+1: